packages/node/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -120,7 +120,7 @@ export default class OidcLoginHandler implements ILoginHandler {
 
     // Construct OIDC Options
     const oidcOptions: IOidcOptions = {
-      issuer: options.oidcIssuer,
+      issuer: issuerConfig.issuer,
```

Comment:
Probably worth a comment here again, just 'cos the reasoning for **not** using the `options.oidcIssuer` is kinda subtle, and could easily be missed when looking at how the options are being set here.

Reply:
Added a comment in b4e1bc67d3ca1e11503629cd605608bff98e44f8